nixos/modules/services/misc/duckling.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.duckling;
+
+  port = cfg.port;
+in {
+  options = {
+    services.duckling = {
+      enable = mkEnableOption "duckling";
+
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/duckling";
+        description = ''
+          Directory to store duckling's logs.
+        '';
+      };
+
+      package = mkOption {
```

Comment:
Are there multiple possible packages that can be used here?

Reply:
It will always be duckling, but someone may want to try out a newer version. Not having this configurable removes this option from users.

Reply:
This is what `overlays` are for.

Reply:
You're right - removed